lua/hbac/init.lua

```diff
@@ -92,7 +92,38 @@ local function close_unused()
 	end
 end
 
+
+-- Create commands to add and remove buffers from the persistant_buffers table
+M.release_buffer = function(bufnr)
+	bufnr = bufnr or vim.api.nvim_get_current_buf()
+	M.persistant_buffers[bufnr] = nil
+end
+
+M.commands = {
+	pin = M.persist_buffer,
+	unpin = M.release_buffer,
+	close_unused = close_unused,
+}
+
+function M.hbac_cmd(arg)
+	if M.commands[arg] then
+		M.commands[arg]()
+		print("Hbac: Buffer " .. arg .. "ned")
+	else
+		print("Unknown Hbac command: ", arg)
+	end
+end
+
+vim.api.nvim_create_user_command("Hbac",
+	function(args)
+		M.hbac_cmd(args.args)
+	end,
+	{
+	nargs = 1,
+})
```

Comment:
```suggestion
	{
	 nargs = 1,
	}
)
```
Do you use a formatter? I'm using the default lsp one, I think indentation is off. I would also put the closing bracket on its own line. It's a nitpick tho 😄

Reply:
Not a nitpick! I'm using `stylua` 0.17.1 and that's what it formatted for me. However you want to do it.

Reply:
Not a nitpick! I'm using `stylua` 0.17.1 and that's what it formatted for me. However you want to do it.

Reply:
I checked and my formatter does not care where I use }) or
```lua
  }
)
```
I just think that the newline version looks better 😄